pages/common/dict.md

```diff
@@ -0,0 +1,24 @@
+# dict
+
+> Client/server software, human language dictionary databases, and tools supporting the DICT protocol.
+> More information: <https://github.com/cheusov/dictd>.
+
+- List available databases:
+
+`dict -D`
+
+- Get database information:
+
+`dict -i {{database_name}}`
+
+- Look up word in database:
```

Comment:
```suggestion
- Look up a word in a specific database:
```

Reply:
That's why I need a dictionary command XD, thank you, @marchersimon.